resources/language/resource.language.de_de/strings.po

```diff
@@ -654,11 +654,11 @@ msgstr ""
 
 msgctxt "#30157"
 msgid "The configuration has been completed"
-msgstr ""
+msgstr "Die Einrichtung wurde abgeschlossen"
 
 msgctxt "#30158"
 msgid "Run add-on configuration wizard"
-msgstr ""
+msgstr "Konfigurationsassistenten für das Addon ausführen"
```

Comment:
@tweimer #30157 and #30158 are translated.

Reply:
Cool. It can be merged now.